optlib/terraform.ctags

```diff
@@ -0,0 +1,28 @@
+#
+# terraform.ctags: regex parse for Terraform (HCL).
+#
+# Author: Antony Southworth <https://github.com/antonysouthworth-halter>
+#
+# This source code is released for free distribution under the terms of the
+# GNU General Public License version 2 or (at your option) any later version.
+#
+# Derived from `vim-terraform-completion`:
+#   - https://github.com/juliosueiras/vim-terraform-completion/blob/master/ctags/terraform.ctags
+#
+# Changed the name from `terraform` to `tf` so vim will recognise it properly based
+# on file extension (*.tf).
+#
+# A notable abscence is `local`, because `locals` are defined inside a block and
+# it's way harder to write a good regex for that.
+#   - https://www.terraform.io/docs/language/values/locals.html
+
+--langdef=tf
```

Comment:
Terraform or TerraformHCL is better. I'm not sure which one is better.
If you are not sure, too, use longer name as I did in "PupppetManifest".